docs/install.rst

```diff
@@ -75,7 +75,7 @@ continuing:
 -  the *Submission Public Key* (from the *Transfer Device*)
 -  the *Submission Key* fingerprint
 -  the email address that will receive alerts from OSSEC
--  the GPG public key and fingerprint for the email address that will
+-  the OSSEC Alert Public Key and fingerprint for the email address that will
```

Comment:
Per our standard convention, we should italicize this term and add a definition to the glossary (happy to talk through what this key is used for).